Birdendra/Auth_Session/login.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready( function(){
+    $('#btn').click( function(){
+        var user_id = document.getElementById("ui").value;
+        var pwd1 = document.getElementById("pwd").value;
+        // console.log(user_id+' '+pwd);
+        
+        $.ajax("http://localhost:58530/api/values",{
```

Comment:
What is this? even after so much training on routes. Why is this values route here? What does it mean?